docs/sources/write/tooling-and-workflows/index.md

```diff
@@ -292,22 +296,22 @@ branch 'my-branch' set up to track 'origin/my-branch'.
 
 The response from GitHub includes a link used to open a pull request for your remote branch.
 Click the link to open GitHub, then click the green button on the upper left of the screen to open the pull request for reviewers.
-Here, you can also edit the title and further detail in the larger text box as well as add reviewers by clicking the Settings icon and entering reviewer GitHub usernames.
+Here, you can also edit the title and further detail in the larger text box as well as add reviewers by clicking the **settings** icon and entering reviewer GitHub usernames.
```

Comment:
Should **settings** be capitalized?

Reply:
Mm, I'm not sure. It's referring to the cog icon in the screenshot. Should it be "cog icon" instead?

<img width="392" alt="Screenshot 2024-04-08 at 16 01 24" src="https://github.com/grafana/writers-toolkit/assets/4599384/4af9bcb1-c7c9-45df-af0b-b8c34a5a58b6">

Reply:
ah, typically this would be the label of the tooltip when you scroll over the icon, but this cog icon doesn't have a tooltip. You could also just click **Reviewers**, which does the same thing as clicking the cog icon?

Reply:
That sounds like a great suggestion

```suggestion
Here, you can also edit the title and further detail in the larger text box as well as add reviewers by clicking the **Reviewers** configuration section and entering reviewer GitHub usernames.
```